PlatformUI/PlatformUI/PlatformListViewModel.swift

```diff
@@ -87,8 +98,8 @@ open class PlatformListViewModel: PlatformViewModeling {
                     Group {
                         let cell =
                         Group {
-                            if item === list.first, let header = self?.header {
-                                header.createView(parentStyle: parentStyle)
+                            if item === list.first && self?.header != nil || item === list.last && self?.footer != nil  {
```

Comment:
I think we need to add parenthesis ... e.g., ( && ) || ( && ).  Otherwise, the right side of the second && won't be evaluated if the left side is true.

Reply:
yep, that is what we want here

![Screenshot 2024-05-24 at 9 25 37 AM](https://github.com/dydxprotocol/v4-native-ios/assets/149746839/2ad42cf8-dd3c-498c-aba7-4aa212ebe54c)


if either `item === list.first && self?.header != nil` is true or `list.last && self?.footer != nil` is true,  we want the body to be reached. I add parenthesis for clarity though